project2/index.html

```diff
@@ -0,0 +1,147 @@
+                            </g>
+                        </g>
+                    </svg></a>
+                <!-- favorite icon -->
+
+                <img src="images/knotless box braids.JPEG" alt=knotless box braids">
+                <svg class="read-more" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60.1 60.1">
+                    <defs>
+                        <style>
+                            .cls-2 {
+                                fill: #fff
+                            }
+                        </style>
+                    </defs>
+                    <g id="Layer_2" data-name="Layer 2">
+                        <g id="Layer_1-2" data-name="Layer 1">
+                            <path d="M30.05 57.1a27.05 27.05 0 1127.05-27 27.07 27.07 0 01-27.05 27z" fill="#eaaa0a" />
+                            <path class="cls-2"
+                                d="M30.05 6A24.05 24.05 0 116 30.05 24.05 24.05 0 0130.05 6m0-6A30.05 30.05 0 1060.1 30.05a30.08 30.08 0 00-30-30z" />
+                            <path class="cls-2"
+                                d="M20.83 27.58l-1.56-1.83h-2.12v1.83h-.87v-5h3.43c1.18 0 2 .56 2 1.6a1.59 1.59 0 01-1.48 1.57l1.63 1.87zM19.65 25c.72 0 1.21-.23 1.21-.86s-.49-.86-1.21-.86h-2.51V25zM28.71 26.82v.76h-5v-5h4.95v.75h-4.11v1.35H28v.76h-3.45v1.42zM34.68 26.36h-3.24l-.64 1.22h-.93l2.73-5h.95l2.72 5h-1zm-.39-.73l-1.23-2.34-1.23 2.34zM43.44 25.06c0 1.55-1.19 2.52-3.06 2.52h-2.56v-5h2.56c1.87-.04 3.06.92 3.06 2.48zm-.88 0c0-1.17-.92-1.77-2.25-1.77h-1.63v3.53h1.63c1.33 0 2.25-.6 2.25-1.76zM21.92 31.18v5h-.87v-3.82l-2.39 2.91h-.05l-2.4-2.91v3.86h-.84v-5h.94l2.34 2.89L21 31.18zM23.81 33.7c0-1.7 1.19-2.7 3.13-2.7s3.13 1 3.13 2.67-1.19 2.68-3.13 2.68-3.13-.98-3.13-2.65zm5.38 0c0-1.22-.88-1.9-2.25-1.9s-2.25.68-2.25 1.9.88 1.91 2.25 1.91 2.25-.68 2.25-1.91zM36.52 36.22L35 34.39h-2.16v1.83H32v-5h3.4c1.17 0 2 .56 2 1.61a1.59 1.59 0 01-1.4 1.52l1.63 1.87zm-1.18-2.57c.72 0 1.21-.23 1.21-.85s-.49-.86-1.21-.86h-2.51v1.71zM44.4 35.47v.75h-5v-5h4.95v.76h-4.11v1.35h3.44V34h-3.44v1.43z" />
+                        </g>
+                    </g>
+                </svg>
+                <!-- read more button -->
+            </div>
+            <h2>hair trends</h2>
+
+            <h3>knotless box braids</h3>
+
+            <h4>chantel harrison</h4>
+            <h4>October 3, 2019</h4>
+
+            <p>This year, braids have been hit with another new wave with the knotless technique taking over. It’s an
+                old
+                technique that is taking over today by storm and has become very popular......</p>
+
+
+            <ul class="social-list">
+                <li><a href="facebook.com"><svg class="facebook" xmlns="http://www.w3.org/2000/svg"
```

Comment:
Even for these demo purposes, I recommend making sure your links work properly (or just using `"#"` everywhere:
```suggestion
                <li><a href="https://facebook.com"><svg class="facebook" xmlns="http://www.w3.org/2000/svg"
```